Add live and soundtrack album types

OzGav opened the pull request:
This is working well locally. If the approach is sound I will add the inference logic to the other providers.

I did a reasonable amount of research on the inference logic and I believe the number of false positives will be very low. In my testing I got none. Even if there is one it just means one album appears in the filtered list which I think is acceptable given the features overall usefulness.

Frontend changes https://github.com/music-assistant/frontend/pull/1138

OzGav committed:
Add live and soundtrack album types

marcelveldt commented on music_assistant/providers/deezer/__init__.py:
please isolate these changes in a dedicated PR

OzGav replied:
I didn’t think it would pass mypy though?

OzGav replied:
done!

OzGav committed:
Remove mypy changes

OzGav committed:
Add infer type if tags missing

OzGav committed:
Fix typos

OzGav committed:
Typo

OzGav committed:
Fix bug normalising WMA tags

WMA tags appear to have a forward slash that also needs to be removed

ffprobe -v quiet -show_entries format_tags -of json "01 Now.wma"
{
    "format": {
        "tags": {
            "album": "Doctor Zhivago",
            "WM/Script": "Latn",
            "track": "1",
            "composer": "Lucy Simon",
            "genre": "Soundtrack",
            "album_artist": "Anthony Warlow",
            "WM/Year": "1995",
            "WM/Media": "CD",
            "MusicBrainz/Album Artist Id": "7ad0a863-e2e0-4038-be9b-02f4011ee348",
            "MusicBrainz/Artist Id": "7ad0a863-e2e0-4038-be9b-02f4011ee348",
            "MusicBrainz/Album Type": "soundtrack",
            "title": "Now",
            "artist": "Anthony Warlow"
        }
    }
}

OzGav committed:
Fix line too long

marcelveldt reviewed (approved):
Thanks, @OzGav !
